include/exadg/fluid_structure_interaction/precice/driver.h

```diff
@@ -83,6 +83,30 @@ class Driver
   std::shared_ptr<ExaDG::preCICE::Adapter<dim, dim, VectorType>> precice;
   ExaDG::preCICE::ConfigurationParameters                        precice_parameters;
 
+  // Nomenclature
+  // time window: denotes time interval after which FSi coupling takes place
+  // subcycling: a single field solver performs several time steps until the next FSI coupling time
+  // window is reached
+  // time step size: refers to single field solvers. In case of subcycling, one
+  // time window consists of several time step sizes.
```

Comment:
If correct, please add the relative clause "... of several time steps, which may have different size for the fluid field and the solid field."